games-arcade/rocksndiamonds/rocksndiamonds-4.0.0.2.recipe

```diff
@@ -48,14 +48,15 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make sdl RW_GAME_DIR=$settingsDir/rocksndiamonds
+	make #RW_GAME_DIR=$settingsDir/rocksndiamonds
```

Comment:
@korli Probably, but as you can see in the above link to the forum, the author intended it to live in a "HOME" enviroment, sending it to $settingsDir is out of my leage (codewise) ...